start-guide/quickstart/manageconnections.rst

```diff
@@ -208,6 +208,32 @@ This step-by-step example shows you how to connect to a device.
 
 .. tip:: Remember - you can put all of these commands into a single script. We'll show you how in the :ref:`parse-output` section. 
 
+
+Depends on your device terminal settings, when you connect to device using CLI and execute command, you would see "press any key to continue". 
+For humans, this point of brake, give possibility to analyse output. However from automation point of view it would break parsers, as they change output data. 
+
+To avoid those, Unicon's (#pyATS connection implementation) is to issue the following on connection established: 
+
+    * no logging console
+    * terminal width 511
+    * etc (depending on platform, which can be specified in testbed file)
+
+.. important:: All these commands affect the terminal behavior, not your device's functionality. 
+
+To disable default configuration:
+
+>>> dev.connect(init_exec_commands=[], init_config_commands=[])
+
+or
+
+>>> dev.connect(mit=True)
+
+For more details, see the topic `customizing your connection <https://pubhub.devnetcloud.com/media/unicon/docs/user_guide/connection.html#customizing-your-connection>`_.
+
```

Comment:
Each plugin can specify their own init commands which are set via the plugin settings. For example, the generic plugin settings can be found here https://github.com/CiscoTestAutomation/unicon.plugins/blob/master/src/unicon/plugins/generic/settings.py#L17